src/Exercism.Analyzers.CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think we should git-ignore these files, right?

Reply:
Yes, I think so. I'll revert it.

Reply:
👍